lms/djangoapps/courseware/grades.py

```diff
@@ -577,7 +598,7 @@ def _progress_summary(student, request, course, field_data_cache=None, scores_cl
 
     max_scores_cache.push_to_remote()
 
-    return chapters
+    return dict(chapters=chapters, scores=locations_to_scores)
```

Comment:
Thinking on this a bit more, can we make this an actual class with attributes instead of a dictionary? That way we might be able to build up some more complex query logic later on.